pom.xml

```diff
@@ -162,14 +165,15 @@
                         <param>edu.ucsb.cs156.*</param>
                     </targetTests>
                     <excludedClasses>
-                        <param>edu.ucsb.cs156.example.aop.LoggingAspect</param>
-                        <param>edu.ucsb.cs156.example.controllers.FrontendController</param>
-                        <param>edu.ucsb.cs156.example.controllers.FrontendProxyController</param>
-                        <param>edu.ucsb.cs156.example.services.CurrentUserServiceImpl</param>
-                        <param>edu.ucsb.cs156.example.ExampleApplication</param>
-                        <param>edu.ucsb.cs156.example.config.SecurityConfig</param>
-                        <param>edu.ucsb.cs156.example.config.SecurityConfig.MyCsrfRequestMatcher</param>
-                        <param>edu.ucsb.cs156.example.config.SpringFoxConfig</param>
+                        <param>edu.ucsb.cs156.happiercows.aop.LoggingAspect</param>
+                        <param>edu.ucsb.cs156.happiercows.controllers.FrontendController</param>
+                        <param>edu.ucsb.cs156.happiercows.controllers.FrontendProxyController</param>
+                        <param>edu.ucsb.cs156.happiercows.services.CurrentUserServiceImpl</param>
+                        <param>edu.ucsb.cs156.happiercows.ExampleApplication</param>
```

Comment:
This might have slipped when migrating from courses search. We have "HappierCowsApplication" which is the equivalent of "ExampleApplication" for courses search, we could remove this line

Reply:
Fixed.. thanks!